proposals/templates/proposals/funding_form.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base/fetc_form_base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block header_title %}
+    {% trans "Informatie over financiering" %} - {{ block.super }}
+{% endblock %}
+
+{% block html_head %}
+    {{ block.super }}
+    <script>
+        $(function () {
+            check_field_required('funding', 'needs_details', 'funding_details', 'proposals');
+            check_field_required('funding', 'needs_name', 'funding_name', 'proposals');
+        });
+    </script>
+{% endblock %}
+
+{% block pre-form-container-content %}
+    {% include "./practice_or_supervisor_warning.html" %}
+{% endblock %}
```

Comment:
Is there any page we would _not_ want to show this warning?

If not, we probably should move this to the form base template